README.md

```diff
@@ -60,3 +60,41 @@ pip install -r dev-requirements.txt
 The `ALLENNLP_VERSION_OVERRIDE` environment variable ensures that the `allennlp` dependency is unpinned so that your local install of `allennlp` will be sufficient. If, however, you haven't installed `allennlp` yet and don't want to manage a local install, just omit this environment variable and `allennlp` will be installed from the master branch on GitHub.
 
 Both `allennlp` and `allennlp-models` are developed and tested side-by-side, so they should be kept up-to-date with each other. If you look at the GitHub Actions [workflow for `allennlp-models`](https://github.com/allenai/allennlp-models/actions), it's always tested against the master branch of `allennlp`. Similarly, `allennlp` is always tested against the master branch of `allennlp-models`.
+
+### Using Docker
+
+Docker provides a virtual machine with everything set up to run AllenNLP--
+whether you will leverage a GPU or just run on a CPU.  Docker provides more
+isolation and consistency, and also makes it easy to distribute your
+environment to a compute cluster.
+
+Once you have [installed Docker](https://docs.docker.com/engine/installation/)
+you can build an image with a specific version of `allennlp` and `allennlp-models` with
+
+   ```bash
+   docker build \
```

Comment:
I would use `docker build -f` instead of `<`.

Reply:
I only used `<` since the image doesn't actually require a context

Reply:
Once you make this adjustment try this command verbatim (copy paste).  You might be missing a dot `.`

Reply:
Hmm I actually think it's more clear to use `<`, i.e. to explicitly not use a context, so that it's obvious that we're not building the image from the local source code.